contracts/examples/hello-world/tests/hello-world.ts

```diff
@@ -23,7 +23,7 @@ describe('hello-world', () => {
     const storeProgram = new Program(storeIdl, CHAINLINK_PROGRAM_ID, provider);
 
     let acIdl = JSON.parse(fs.readFileSync('../../target/idl/access_controller.json'));    
-    const accessControllerProgram = new Program(acIdl, "2F5NEkMnCRkmahEAcQfTQcZv1xtGgrWFfjENtTwHLuKg", provider);
+    const accessControllerProgram = new Program(acIdl, "DzzjdPWNfwHZmzPVxnmqkkMJraYQQRCpgFZajqkqmU6G", provider);
```

Comment:
We could also extract this as a declared constant.

Reply:
+1